packages/mux-uploader/src/mux-uploader-progress.ts

```diff
@@ -6,10 +6,11 @@ import { formatProgress } from './utils/progress';
 const template = document.createElement('template');
 const ariaDescription = 'Media upload progress bar';
 
-template.innerHTML = `
+template.innerHTML = /*html*/ `
 <style>
   :host {
     position: relative;
+    display: contents;
```

Comment:
I would try to avoid setting a custom element to `display: contents`.
it's surprising the host would not have dimensions (box)